### DISCUSSION TITLE
Proposed Working Sessions in Vancouver - week of Sept 12th, 2022

### DISCUSSION THREAD

**Discussion by bmann** [ipvm-wg/spec#4] [Community Call]
We're looking at the week of September 12th to gather in Vancouver for some in person sessions.

Working on details in this HackMD https://hackmd.io/sDgZBt9eSNqyHylW6qTLXA

**Comment by RangerMauve** [comment]
:o Is this going to be recorded by any chance?

**Reply by bmann** [reply]
@RangerMauve if there are "presentation" style things, it will get recorded, but this is more likely to be whiteboard / flip chart small group discussion. First one of these, so we'll see how it goes (and yes, tight timing).

**Reply by RangerMauve** [reply]
Hmmm, I'd love to attend on behalf of the IPLD team. 🤔 I'll see if I could get PL to ship me over there. 😁

**Comment by bmann** [comment]
OK, we've run out of time with this which was already aggressive to begin with. We'll focus on sessions during PL Lab Week / IPFS Camp / etc in Lisbon end of October --> https://github.com/ipvm-wg/spec/discussions/5

cc @RangerMauve
